tools/wpt/utils.py

```diff
@@ -9,19 +9,13 @@
 import zipfile
 from io import BytesIO
 from socket import error as SocketError  # NOQA: N812
+from typing import Any, Callable, Dict
 from urllib.request import urlopen
 
-MYPY = False
-if MYPY:
-    from typing import Any
-    from typing import Callable
-    from typing import Dict
-
 logger = logging.getLogger(__name__)
 
 
-class Kwargs(dict):
-    # type: Dict[Any, Any]
+class Kwargs(Dict[Any, Any]):
```

Comment:
This change was necessary because the type annotation was on the wrong line, and actually it shouldn't be `class Kwargs(dict: Dict[Any, Any])` either, because inheritance is from something concrete, it's not an argument or somesuch. This is the way to do it AFAICT from examples I've read.